lib/logstash/outputs/scalyr.rb

```diff
@@ -749,7 +765,16 @@ def create_multi_event_request(scalyr_events, logstash_events, current_threads,
 
     # We time serialization to get some insight on how long it takes to serialize the request body
     start_time = Time.now.to_f
-    serialized_body = body.to_json
+    begin
+      serialized_body = body.to_json
+    rescue Java::JavaLang::ClassCastException => e
+      @logger.warn("Error serializing events to JSON, likely due to the presence of Bignum values. Converting Bignum values to strings.")
+      @stats_lock.synchronize do
+        @multi_receive_statistics[:total_bignum_json_errors] += 1
```

Comment:
Can we please add another counter which counts total conversions (aka also successful conversions and not just errors).